plugins/woocommerce/package.json

```diff
@@ -33,7 +33,7 @@
 		"env:test": "pnpm env:dev",
 		"env:perf": "pnpm env:dev && pnpm env:performance-init",
 		"preinstall": "npx only-allow pnpm",
-		"postinstall": "rimraf -g vendor/woocommerce && XDEBUG_MODE=off composer install --quiet",
+		"postinstall": "rimraf -g vendor/woocommerce packages/email-editor && XDEBUG_MODE=off composer install --quiet",
```

Comment:
@triple0t The title and the description of this Pull Request only talk about the blueprint. However, this doesn't seem to be related. Is this a forgotten change, or is it intentional?

Reply:
@pavel-mailpoet Yes, this is intentional. 
This ensures we have a fresh version of the email editor package every time pnpm install is run.